README.md

```diff
@@ -96,7 +96,7 @@ For an overview of the project's architecture please check out our [ARCHITECTURE
 
 ---------------
 
-For a deep dive in Unity Netcode and Boss Room, visit our [doc](https://docs-multiplayer.unity3d.com/) and [Unity Multiplayer Resources](https://docs-multiplayer.unity3d.com/netcode/current/learn/introduction) sections.
+For a deep dive in Unity Netcode and Boss Room, visit our [docs site](https://docs-multiplayer.unity3d.com/).
```

Comment:
We still do have a ressource section though. It's called concepts and faqs now.
https://docs-multiplayer.unity3d.com/netcode/current/reference/glossary/high-level-terminology